### PR TITLE
[excel] (Formatting) Conditional formatting and user input sample

### PR DESCRIPTION
**Pull request by AlexJerabek**
This sample shows how to make conditional formatting using relative references. It also has parameters to allow for user input.

### CODE REVIEW

**Comment by michelleranmsft** [comment]
It seems like this script breaks if there's a ! in the sheet name - are we concerned about that case?

**Comment by AlexJerabek** [comment]
> It seems like this script breaks if there's a ! in the sheet name - are we concerned about that case?

Probably. I believe switching the string parsing to look for the lastIndexOf a "!" should fix that. Worked when I tested it, but let me know if you think of other corner cases.

**Review by michelleranmsft — APPROVED** [review]

File: docs/resources/samples/conditional-formatting-parameters.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+# Set conditional formatting for cross-column comparisons
+
+This sample shows how to apply conditional formatting to a range. The conditions used are comparing values to those in an adjacent column.  Additionally, this sample uses [parameters to get user input](../../develop/user-input.md). This lets the person running the script select the range, the type of comparison, and the colors.
+
+## Sample code: Set conditional formatting
+
+```TypeScript
+/**
+ * Format a range on the current sheet based on values in an adjacent column.
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/samples/conditional-formatting-parameters.md:16]:
```suggestion
 * Formats a range on the current sheet based on values in an adjacent column.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Comment by AlexJerabek** [comment]
Thanks @michelleranmsft and @ElizabethSamuel-MSFT. I'll merge when the parameter feature rolls out next week.

**Pull request merged**